tests/Generic.Repository.Test/Cache/CacheRepositoryIOAccessTest.cs

```diff
@@ -69,57 +71,57 @@ void CheckIfIsValid(object result)
+                        for (var i = 0; i <= MaxIterations; i++)
                         {
-                            var tarefa = Task.Run(async () =>
+                            var @task = Task.Run(async () =>
                             {
-                                var result = await Cache.GetMethodGet(NameType, NameProperty, default);
+                                var result = await Cache.GetMethodGet(NameType, NameProperty, default).ConfigureAwait(false);
                                 CheckIfIsValid(result);
                             });
-                            listaTarefaIO.Add(tarefa);
+                            listTaskIO.Add(@task);
 
-                            tarefa = Task.Run(async () =>
+                            @task = Task.Run(async () =>
                             {
-                                var result = await Cache.GetMethodSet(NameType, NameProperty, default);
+                                var result = await Cache.GetMethodSet(NameType, NameProperty, default).ConfigureAwait(false);
                                 CheckIfIsValid(result);
                             });
-                            listaTarefaIO.Add(tarefa);
+                            listTaskIO.Add(@task);
 
-                            tarefa = Task.Run(async () =>
+                            @task = Task.Run(async () =>
                             {
-                                var result = await Cache.GetProperty(NameType, NameProperty, default);
+                                var result = await Cache.GetProperty(NameType, NameProperty, default).ConfigureAwait(false);
                                 CheckIfIsValid(result);
                             });
-                            listaTarefaIO.Add(tarefa);
+                            listTaskIO.Add(@task);
 
-                            tarefa = Task.Run(async () =>
+                            @task = Task.Run(async () =>
                             {
-                                var result = await Cache.GetAttribute(NameType, NameProperty, NameAttribute, default);
+                                var result = await Cache.GetAttribute(NameType, NameProperty, NameAttribute, default).ConfigureAwait(false);
                                 CheckIfIsValid(result);
                             });
-                            listaTarefaIO.Add(tarefa);
+                            listTaskIO.Add(@task);
                         }
 
-                        await Task.WhenAll(listaTarefaIO);
+                        await Task.WhenAll(listTaskIO);
                     });
 
-                    listaTarefas.Add(tarefaGeral);
+                    scheduledListTask.Add(mainTask);
                 }
                 catch (Exception)
                 {
                     valid = false;
                 }
 
-                await Task.WhenAll(listaTarefas);
+                await Task.WhenAll(scheduledListTask);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/guilhermecaixeta/Generic.Service.DotNetCore2.2/pullRequest?prid=4961311)